### PR TITLE
[docs]: Fix documentation for the Date-Range Picker Component

### PR DESCRIPTION
**Pull request by rudrakshnalbalwar**
🔧 Documentation Update for Date-Range Picker Component 🗓️
This PR addresses several improvements in the date-range-picker.md documentation to enhance readability and consistency which have been mentioned in issue #11011 . Below is a summary of the changes:

Formatting Updates:

Replaced Fx with fx (bold and lowercase) for consistency 🔄.
Removed padding-bottom: 24px from h2 headers to maintain consistent spacing 🎨.
Content Updates:

Replaced the term "widget" with "component" to align with current terminology across the document 🖥️.
Updated the "On select" event description for better sentence structure and clarity ✍️.
Added missing full stops at the end of descriptions in the Properties and Events tables for better formatting 📑.
Removed unnecessary description under the "Events" heading to streamline the section 🚮.
Added a horizontal divider before the "Styles" section for better visual separation 📏.
Documentation Structure:

Ensured the file adheres to markdown and JSX conventions for proper rendering on the ToolJet platform ✅.

### CODE REVIEW

**Comment by rudrakshnalbalwar** [comment]
@PriteshKiri I had made the changes as you mentioned. You can check it and let me know if any change has to be made. Thank you!!

File: docs/versioned_docs/version-2.50.0-LTS/widgets/date-range-picker.md [hunk 1]
```diff
@@ -4,37 +4,35 @@ title: Date-range Picker
...
 :::info
 Check [Action Reference](/docs/category/actions-reference) docs to get the detailed information about all the **Actions**.
 :::
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Component Specific Actions (CSA)
```
Comment (RudraPratik30) [inline docs/versioned_docs/version-2.50.0-LTS/widgets/date-range-picker.md:38]: Hey @rudrakshnalbalwar, here on line no. 43 you need to remove the paddingBottom

File: docs/versioned_docs/version-2.50.0-LTS/widgets/date-range-picker.md [hunk 2]
```diff
@@ -48,45 +46,47 @@ There are currently no CSA (Component-Specific Actions) implemented to regulate
...
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Border radius | This is to modify the border radius of the date range picker. The field expects only numerical value from `1` to `100`| By default, it's set to `0`|
-| Visibility | This is to control the visibility of the widget. If `{{false}}` the widget will not visible after the app is deployed. It can only have boolean values i.e. either `{{true}}` or `{{false}}`. | By default, it's set to `{{true}}`|
-| Disable | This property only accepts boolean values. If set to `{{true}}`, the widget will be locked and becomes non-functional. | By default, its value is set to `{{false}}` |
+| Border radius | This is to modify the border radius of the date range picker. The field expects only numerical value from `1` to `100`. | By default, it's set to `0`. |
+| Visibility | This is to control the visibility of the widget. If `{{false}}` the component will not visible after the app is deployed. It can only have boolean values i.e. either `{{true}}` or `{{false}}`. | By default, it's set to `{{true}}`. |
```
Comment (RudraPratik30) [inline docs/versioned_docs/version-2.50.0-LTS/widgets/date-range-picker.md:85]: Here you need to update the widget to the component.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @rudrakshnalbalwar please make the mentioned changes.

**Comment by rudrakshnalbalwar** [comment]
Hey @PriteshKiri 
I already made the changes requested by @RudraPratik30  can you please review it and let me know if any further change has to be made

**Comment by RudraPratik30** [comment]
Hey @rudrakshnalbalwar, thanks for contributing!
The changes look good to me and forwarding it to the team for final review.
You can expect your PR to be merged by the end of this week, thanks again!

**Review by RudraPratik30 — DISMISSED** [review]

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Pull request merged**